schemas/bagel_schema.json

```diff
@@ -0,0 +1,81 @@
+{
+    "GLOBAL_COLUMNS": {
+        "participant_id": {
+            "Description": "Participant identifier within a given dataset.", 
+            "dtype": "str",
+            "IsRequired": true,
+            "IsPrefixedColumn": false
+        },
+        "session": {
+            "Description": "Participant session ID.", 
+            "dtype": "str",
+            "IsRequired": true,
+            "IsPrefixedColumn": false
+        },
+        "has_mri_data": {
+            "Description": "Whether or not participant had MRI data acquired in a given session.",
+            "dtype": "bool",
+            "IsRequired": true,
+            "Range": [true, false],
+            "IsPrefixedColumn": false
+        },
+        "Datatype_": {
```

Comment:
@nikhil153 I understand! I wonder though if it would be useful to have some way to visually distinguish columns in bagel.csv which have a schema-defined prefix and user-defined suffix, vs. column names that are expected by the schema but have underscores in them.

e.g. 
`participant_id` is the full name of a column expected by the schema - `participant_id_blah` would not be recognized.
vs.
`Datatype_anat` and `Datatype_func` would both be acceptable, since whatever comes after the underscore doesn't matter - the dashboard just looks for "Datatype_"

So far in the schema I've attempted to distinguish these two column types by capitalizing all column names that have an attribute `"IsPrefixedColumn": true`, e.g. Stage_, Phase_, Datatype_, and having all other column names lowercase. I wonder though if this still isn't explicit enough - alternatively, I could capitalize the entire prefix for prefixed columns only (STAGE_, PHASE_ DATATYPE_), or have _two_ separating underscores as a case-insensitive option (Stage__, Phase__, Datatype__).

Reply:
@alyssadai I think it makes sense to differentiate the two cases. I like the capitalized prefix idea: it's a lot more obvious and seems more deliberate than just capitalizing the first letter.